.github/workflows/ci.yml

```diff
@@ -39,26 +42,23 @@ jobs:
               sbt -v ++${{ matrix.scala }} scalafmtCheck scalafmtSbtCheck scala_2_11/test:compile scala_2_11/test:doc
               sbt -v ++${{ matrix.scala }} scala_2_11/test
               ;;
-            2.12.14)
+            2.12.16)
               sbt -v ++${{ matrix.scala }} test:compile test:doc
               sbt -v ++${{ matrix.scala }} coverage test coverageReport
               sbt -v ++${{ matrix.scala }} coverageAggregate
               ;;
-            2.13.6)
+            *)
               sbt -v ++${{ matrix.scala }} test:compile test:doc
               sbt -v ++${{ matrix.scala }} test
               ;;
-            *)
```

Comment:
No to fail there, SBT will anyway check whether the version selected by the CI is supported (and it avoid to hard code more cases)